tests/components/RangeInput.test.tsx

```diff
@@ -0,0 +1,296 @@
+import userEvent from '@testing-library/user-event';
+import { State } from '@yext/answers-headless-react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { RangeInput } from '../../src/components/Filters';
+import { mockAnswersHooks, spyOnActions } from '../__utils__/mocks';
+import { Matcher, SelectableFilter } from '@yext/answers-headless-react';
+import { FiltersContext, FiltersContextType } from '../../src/components/Filters/FiltersContext';
+import { FilterGroupContext, FilterGroupContextType } from '../../src/components/Filters/FilterGroupContext';
+
+const mockedState: Partial<State> = {
+  filters: {
+    static: [],
+  },
+  meta: {
+    searchType: 'vertical'
+  }
+};
+
+const mockedActions = {
+  state: mockedState,
+  setOffset: jest.fn(),
+  setFilterOption: jest.fn(),
+  resetFacets: jest.fn(),
+  executeVerticalQuery: jest.fn()
+};
+
+const selectableFilter: SelectableFilter = {
+  selected: true,
+  fieldId: '123',
+  matcher: Matcher.Between,
+  value: 'test'
+};
+
+const filterContextValue: FiltersContextType = {
+  selectFilter: () => null,
+  applyFilters: () => null,
+  filters: []
+};
+
+const filterContextValueDisabled: FiltersContextType = {
+  selectFilter: () => null,
+  applyFilters: () => null,
+  filters: [selectableFilter]
+};
+
+const filterGroupContextValue: FilterGroupContextType = {
```

Comment:
Is it possible to have this be `Partial<FilterGroupContextType>` and exclude the null values? not sure if typescript would yell at you

Reply:
Ah ok keep them then!

Reply:
We're not using ts-jest so when jest runs it ignores the typescript errors